osu.Game/Graphics/UserInterface/ScreenTitle.cs

```diff
@@ -41,6 +49,11 @@ public Color4 AccentColour
             set => pageText.Colour = value;
         }
 
+        protected virtual Drawable CreateIcon() => iconSprite = new SpriteIcon
```

Comment:
maybe move below ctor?

Reply:
Is this really where we want to be setting `iconSprite`? What if a deriver calls this method multiple times?

Reply:
then they are silly

Reply:
what do you mean?